debian/install

```diff
@@ -0,0 +1 @@
+automysqlbackup usr/local/bin
```

Comment:
I would not use `usr/local/bin`. I'd like to have everything packaged directly under `/usr`. `/usr/local` would only be used for stuff coming from elsewhere (ideally, only Puppet). Once the package is built, we'll update the path in Puppet.